nixos/roles/k3s/server.nix

```diff
@@ -304,12 +304,32 @@ in {
       ensureDatabases = [ "kubernetes" ];
       ensureUsers = [ {
         name = "root";
-        ensurePermissions = {
-          "DATABASE kubernetes" = "ALL PRIVILEGES";
-        };
       } ];
     };
 
+    systemd.services.kubernetes-ensure-db-permissions = {
```

Comment:
Is there a reason why to use a custom systemd service instead of `services.postgresql.ensureUsers.*.ensureDBOwnership`?